opengrok-tools/setup.py

```diff
@@ -0,0 +1,58 @@
+import os
+import unittest
+from setuptools import setup
+
+
+def readme():
+    with open(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'src', 'main', 'python', 'README.md'), 'r') as readme:
+        return readme.read()
+
+
+def my_test_suite():
+    test_loader = unittest.TestLoader()
+    test_suite = test_loader.discover(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'src', 'test', 'python'), pattern='test_*.py')
+    return test_suite
+
+setup(
+    name='opengrok-tools',
+    version='1.1rc48',
+    packages=[
+        'opengrok_tools',
+        'all',
+        'all.utils',
+        'all.scm',
+    ],
+    package_dir={
+        'opengrok_tools': 'src/main/python/opengrok_tools',
+        'all': 'src/main/python/opengrok_tools/all',
+        'all.scm': 'src/main/python/opengrok_tools/all/scm',
+        'all.utils': 'src/main/python/opengrok_tools/all/utils',
+    },
+    url='https://github.com/OpenGrok/OpenGrok',
+    license='CDDL',
+    author='Oracle',
+    author_email='opengrok-users@yahoogroups.com',
```

Comment:
`opengrok-dev` would be more fitting I think

Reply:
done